src/Symfony/Component/Messenger/EventListener/SendFailedMessageToFailureTransportListener.php

```diff
@@ -40,6 +48,11 @@ public function onMessageFailed(WorkerMessageFailedEvent $event)
             return;
         }
 
+        $hasFailureTransports = $this->hasFailureTransports($event);
+        if (!$hasFailureTransports) {
+            return;
+        }
```

Comment:
Is this quite right? I looks like `hasFailureTransports()` doesn't use the `$event` argument. Should we instead use the code below around like 69-70 to covers this? We could make `$this->getFailureSender` use `$this->failureSenders->has()` and return `null` if none is fond.